test/helpers/movies_helper_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class MoviesHelperTest < ActionView::TestCase
+  def setup
+    @movie_no_reviews = create(:movie, title: 'movie_no_review')
+    @movie_with_reviews = create(:movie, title: 'movie_with_reviews')
+    @user = create(:user)
+    @review = create(:review, movie: @movie_with_reviews, user: @user, stars: 5)
+  end
+
+  def test_movie_no_reviews
+    result = average_stars(@movie_no_reviews)
+    assert_includes result, 'No reviews'
+  end
```

Comment:
Hmmmm, maybe to be consistent change here to
```ruby
test "movie_no_reviews" do
...
end
```
Or in the other tests to method syntax.